packages/sdk/shell/src/components/HuePicker.tsx

```diff
@@ -25,7 +34,86 @@ export type HuePickerProps = {
   onClickClear?: ButtonProps['onClick'];
 };
 
-export const HuePicker = ({ disabled, hue, onChangeHue, defaultHue, onClickClear }: HuePickerProps) => {
+export const HuePickerToolbarButton = ({
```

Comment:
could do with jsdoc helping to understand the difference between the two pickers and when each should be used (for emoji too)